src/composablesTs/useAutoPosition.ts

```diff
@@ -1,77 +1,66 @@
 import { computed, toValue, ref } from "vue";
 import { isSSR } from "../utilsTs/utilHelper.ts";
 
-import type { Ref, ComputedRef } from "vue";
-
-interface PositionXY {
-  x: Position;
-  y: Position;
-}
-
-export enum Position {
-  Left = "left",
-  Right = "right",
-  Top = "top",
-  Bottom = "bottom",
-  Auto = "auto",
+import type { ComputedRef, MaybeRef, Reactive } from "vue";
+
+export enum POSITION {
+  left = "left",
```

Comment:
Enum name should be PascalCase
Enums keys should be UPPER_CASE or PascalCase related to the TS conventions.
I prefer PascalCase.

follow ts name convention for enums: PascalCase and for enum keys PascalCase
https://aykhanhuseyn.medium.com/typescript-naming-conventions-crafting-maintainable-code-7d872234fe17